website/source/docs/connect/registration/service-registration.html.md

```diff
@@ -81,7 +81,8 @@ registering a proxy instance.
     "local_service_address": "127.0.0.1",
     "local_service_port": 9090,
     "config": {},
-    "upstreams": []
+    "upstreams": [],
```

Comment:
GH won't let me comment on the first line of this file, but we may want to change the name of the doc to something like "Sidecar Proxy Standalone Registration" to distinguish it from a gateway service registration.